.gitignore

```diff
@@ -92,4 +92,7 @@ typings/
 # DynamoDB Local files
 .dynamodb/
 
+# package-lock.json
+package-lock.json
```

Comment:
Hey @drayzii Please let us keep the `package-lock.json` file in our project because it will help us keep the versions tree of the project dependencies. So let us not ignore it.